spotinst/version.go

```diff
@@ -1,7 +1,7 @@
 package spotinst
 
 // SDKVersion is the current version of the SDK.
-const SDKVersion = "1.61.0"
+const SDKVersion = "1.62.0"
```

Comment:
```suggestion
const SDKVersion = "1.61.0"
```

The version number is bumped for every new release as part of the release process.